app/services/flow/flows/provider_dependants.rb

```diff
@@ -19,7 +19,7 @@ class ProviderDependants < FlowSteps
                 :dependants
               end
             else
-              application.checking_non_passported_means? ? :means_summaries : :check_passported_answers
+              :check_income_answers
```

Comment:
Is there a reason this conditional is no longer needed?

Reply:
Yep, so this check was previously redundant since it was essentially checking whether the application is passported and therefore needed to go back to the check_passported_answers page instead of the means summary page, but dependants are part of the income section (i.e. only appear in non passported) so they would never have come from the passported check answers page anyway. Also it's now even more redundant because dependants answers will be checked on the new check income answers page which is again, only on the non-passported journey